x-pack/plugins/monitoring/public/components/metricbeat_migration/instruction_steps/elasticsearch/enable_metricbeat_instructions.js

```diff
@@ -0,0 +1,227 @@
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+import { i18n } from '@kbn/i18n';
+import React, { Fragment } from 'react';
+import {
+  EuiSpacer,
+  EuiCodeBlock,
+  EuiLink,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiButton,
+  EuiCallOut,
+  EuiText
+} from '@elastic/eui';
+import { Monospace } from '../components/monospace';
+import { FormattedMessage } from '@kbn/i18n/react';
+import { statusTitle } from './common_elasticsearch_instructions';
+
+export function getElasticsearchInstructionsForEnablingMetricbeat(product, _meta, {
+  esMonitoringUrl,
+  hasCheckedStatus,
+  checkingMigrationStatus,
+  checkForMigrationStatus,
+  autoCheckIntervalInMs
+}) {
+  const installMetricbeatStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatTitle', {
+      defaultMessage: 'Install Metricbeat on the same server as Elasticsearch'
+    }),
+    children: (
+      <EuiText>
+        <p>
+          <FormattedMessage
+            id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatDescription"
+            defaultMessage="Follow {link}."
+            values={{
+              link: (
+                <EuiLink href="https://www.elastic.co/guide/en/beats/metricbeat/current/metricbeat-installation.html" target="_blank">
+                  <FormattedMessage
+                    id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.installMetricbeatLinkText"
+                    defaultMessage="the instructions here"
+                  />
+                </EuiLink>
+              )
+            }}
+          />
+        </p>
+      </EuiText>
+    )
+  };
+
+
+  const enableMetricbeatModuleStep = {
+    title: i18n.translate('xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleTitle', {
+      defaultMessage: 'Enable and configure the Elasticsearch module in Metricbeat'
+    }),
+    children: (
+      <Fragment>
+        <EuiCodeBlock
+          isCopyable
+          language="bash"
+        >
+            metricbeat modules enable elasticsearch-xpack
+        </EuiCodeBlock>
+        <EuiSpacer size="s"/>
+        <EuiText>
+          <p>
+            <FormattedMessage
+              id="xpack.monitoring.metricbeatMigration.elasticsearchInstructions.enableMetricbeatModuleDescription"
+              defaultMessage="By default the module will collect Elasticsearch monitoring metrics from {url}.
+              If the local Elasticsearch server has a different address,
+              you must specify it via the hosts setting in the {module} file."
+              values={{
+                module: (
+                  <Monospace>modules.d/elasticsearch-xpack.yml</Monospace>
+                ),
+                url: (
+                  <Monospace>http://localhost:9200</Monospace>
+                )
```

Comment:
Per https://www.elastic.co/guide/en/elasticsearch/reference/master/configuring-metricbeat.html (step 5) there's also an extra step here if security is enabled in the cluster.

Reply:
Do you think we should just copy and paste the step exactly here, including the security setup and the links (like the link for `remote_monitoring_collector built-in role`)?

Reply:
I think that with security features in basic licenses now it's more and more likely that users will need to take those extra configuration steps.  So at the very least I'd say something like, "If security features are enabled <on whichever cluster this step pertains to>, you must also <add user and password information, etc>. For more information, see <link to docs>". If you want to put the full set of steps in and need my helping drafting or review, I'm happy to help.